runtime/vm/jnimisc.cpp

```diff
@@ -394,13 +394,8 @@ getObjectClass(JNIEnv *env, jobject obj)
 jint JNICALL
 getVersion(JNIEnv *env)
 {
-	J9VMThread *currentThread = (J9VMThread*)env;
-	J9JavaVM *vm = currentThread->javaVM;
-
-	if (J2SE_VERSION(vm) >= J2SE_V10) {
+	if (J2SE_VERSION(((J9VMThread*)env)->javaVM) >= J2SE_V11) {
 		return JNI_VERSION_10;
```

Comment:
This needs to be checked, should it be 11?

Reply:
`JDK11` still has `JNI_VERSION_10`. By the way, I didn't update JVMTI version for `JDK11` and leave it to JEP331 PR instead.